src/index.js

```diff
@@ -1,32 +1,121 @@
 // 1. Write a function that accepts an array of numbers. Return a new array with all elements doubled
 // Ex: double([1,2,3]) => [2,4,6]
 
+const numbers = [1,2,3];
+const newArr = numbers.map((num)=>{return num*2})
+console.log(newArr)
+
 // 2. Write a function that accepts an array of numbers. Return a new array with only elements that are even number
 // Ex: filterEven([1,2,3,4,5,6]) => [2,4,6]
 
-// 3. Write a function that accepts an array of student object. Each student object will have a name property. Return those students that has name length larger than 5
+//another way of doing
+// const numbers = [1,2,3,4,5,6]
+// const newArr = numbers.filter(filterEven)
+// console.log(newArr)
+// function filterEven(num){
+//   return !(num%2)
+// }
+
+const numbers1 = [1,2,3,4,5,6]
+const newArr1 = numbers1.filter((num)=> {return !(num % 2)})
+console.log(newArr1)
+
+
+// 3. Write a function that accepts an array of student object. Each student object will have a 
+//name property. Return those students that has name length larger than 5
 // Ex: const students = [{name: "abc"}, {name: "Umer Khan"}, {name: "Duy Nguyen"}]
 //     filterStudent(students) => [{name: "Umer Khan"}, {name: "Duy Nguyen"}]
 
+const students = [{name: "abc"}, {name: "Umer Khan"}, {name: "Duy Nguyen"}]
+const filterStudent = students.filter((student) => {
+  return  student.name.length > 5;
+})
+console.log(filterStudent)
+
+
 // 4. Write a function that accepts an unlimited amount of arguments. The arguments are of type number. Return the largest number
 // Ex: largest(1,2,3,4,5,6) => 6
 // Hint: use Math.max()
 
-// 5. Write a function that accepts an unlimited amount of arguments. The arguments are of type number. Return the sum of all of them
+const largest = (...allArgs) => {return Math.max(...allArgs)}
```

Comment:
better to use reduce .... to be familiar with it

Reply:
The better way is `Math.max`, but yes challenge yourself with reduce 😄